streams/src/test/java/org/apache/kafka/streams/integration/utils/KafkaEmbedded.java

```diff
@@ -91,6 +91,7 @@ private Properties effectiveConfigFrom(final Properties initialConfig) {
         effectiveConfig.put(KafkaConfig.MessageMaxBytesProp(), 1000000);
         effectiveConfig.put(KafkaConfig.ControlledShutdownEnableProp(), true);
         effectiveConfig.put(KafkaConfig.ZkSessionTimeoutMsProp(), 10000);
+        effectiveConfig.put(KafkaConfig.RackProp(), "rack0");
```

Comment:
why do we need this overwrite?

Reply:
Good catch. Might be some left over change. Let me delete it